mgmt/RecordsConfig.cc

```diff
@@ -365,12 +365,17 @@ static const RecordElement RecordsConfig[] =
   //        # verbose via string
   //        #
   //        # 0 - no extra info added to string
-  //        # 1 - all extra information added
-  //        # 2 - some extra info added
+  //        # 1 - minimal information.
+  //        # 2 - more information
+  //        # 3 - maximum information.
   {RECT_CONFIG, "proxy.config.http.request_via_str", RECD_STRING, "ApacheTrafficServer/" PACKAGE_VERSION, RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
   {RECT_CONFIG, "proxy.config.http.response_via_str", RECD_STRING, "ApacheTrafficServer/" PACKAGE_VERSION, RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http.request_via_transport", RECD_STRING, "compact", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
+  ,
```

Comment:
I'm very torn on this... All (well, almost all) of our configs do not use this sort of semantics. As much as it looks nicer, it seems very non-standard, and maybe even confusing. Whereas other configs would do e.g. 0|1|2, just like the existing Via configs, we now introduce a completely new format / syntax of configuration values.

At a minimum, I think this ought to be discussed, including a plan of making a migration (and new standard) for semantic configurations instead of "int" values.